Firmware/Marlin_main.cpp

```diff
@@ -11043,8 +11042,7 @@ void restore_print_from_eeprom(bool mbl_was_active) {
 	sprintf_P(cmd, PSTR("M220 S%d"), feedmultiply_rec);
 	enquecommand(cmd);
   // Set the fan speed saved at the power panic.
-	strcpy_P(cmd, PSTR("M106 S"));
-	strcat(cmd, itostr3(int(fan_speed_rec)));
+	sprintf_P(cmd, PSTR("M106 S%c"), fan_speed_rec);
```

Comment:
This looks wrong. %c is used for printing ascii characters, which is not the intended case here. Use %u instead. I know it's unfortunate, but on the AVR, all function parameters get blown up to at least 2B anyway, so it's not like there is a more efficient option. %hhu also doesn't work unfortunately.

Reply:
Fixed 👍